templates/rally-zero-one/measurements/measurements.1.schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "Please do not edit this schema by hand. Generate it instead by running npm run build:schema.",
```

Comment:
Is this generated from rally?

Reply:
yes, this was from our end.

Reply:
Maybe it would be good to document more clearly here where this came from. Just a link to the repo should be fine (unless it can be installed via npm?)

Reply:
Added to the title